api-schema/src/main/java/org/stellar/anchor/api/custody/fireblocks/TransactionSubStatus.java

```diff
@@ -0,0 +1,81 @@
+package org.stellar.anchor.api.custody.fireblocks;
+
+import com.google.gson.annotations.SerializedName;
+
+public enum TransactionSubStatus {
+  INSUFFICIENT_FUNDS,
+  AMOUNT_TOO_SMALL,
+  UNSUPPORTED_ASSET,
+  UNAUTHORISED__MISSING_PERMISSION,
+  INVALID_SIGNATURE,
+  API_INVALID_SIGNATURE,
+  UNAUTHORISED__MISSING_CREDENTIALS,
+  UNAUTHORISED__USER,
+  UNAUTHORISED__DEVICE,
+  INVALID_UNMANAGED_WALLET,
+  INVALID_EXCHANGE_ACCOUNT,
+  INSUFFICIENT_FUNDS_FOR_FEE,
+  INVALID_ADDRESS,
+  WITHDRAW_LIMIT,
+  API_CALL_LIMIT,
+  ADDRESS_NOT_WHITELISTED,
+  TIMEOUT,
+  CONNECTIVITY_ERROR,
+  THIRD_PARTY_INTERNAL_ERROR,
+  CANCELLED_EXTERNALLY,
+  INVALID_THIRD_PARTY_RESPONSE,
+  VAULT_WALLET_NOT_READY,
+  MISSING_DEPOSIT_ADDRESS,
+  ONE_TIME_ADDRESS_DISABLED,
+  INTERNAL_ERROR,
+  UNKNOWN_ERROR,
+  AUTHORIZER_NOT_FOUND,
+  INSUFFICIENT_RESERVED_FUNDING,
+  MANUAL_DEPOSIT_ADDRESS_REQUIRED,
+  INVALID_FEE,
+  ERROR_UNSUPPORTED_TRANSACTION_TYPE,
+  UNSUPPORTED_OPERATION,
+  @SerializedName("3RD_PARTY_PROCESSING")
```

Comment:
❓ Are these `@SerializedName` annotations necessary if the field name is the same as the serialized name?

Reply:
From Fireblocks we may receive a sub status `3RD_PARTY_PROCESSING` but in Java we can't specify a field name which starts with a number. That's why, we changed field name to `THIRD_PARTY_PROCESSING`.

Field name is not the same as serialized name so we had to add `@SerializedName` to match value to corresponding field in enum.